projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -93,7 +91,18 @@ export class ExplorerComponent {
   private readonly explorerDashboardBuilder: ExplorerDashboardBuilder;
   public readonly resultsDashboard$: Observable<ExplorerGeneratedDashboard>;
   public readonly vizDashboard$: Observable<ExplorerGeneratedDashboard>;
-  public readonly contextLabel$: Observable<ContextLabel>;
+
+  public contextItems: ToggleItem<ContextLabel>[] = [
+    {
+      label: ExplorerComponent.API_TRACES,
```

Comment:
We convert back aand forth between theses labels and the associated values because the toggle-button didn't support a distinct value - we should fix that here (Removing `ContextLabel` the type should give a decent how-to guide on this)